.changeset/violet-rings-prove.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/text-area': minor
```

Comment:
We don't need to do a beta release of this component – it's pretty straightforward.

Reply:
I asked Rob if he wanted to beta it because he hasn't had a chance to do an audit of the texarea component, so wasn't sure if the styles would change a little more significantly at some point

Reply:
The API is unlikely to change, as is the design (given how similar the component is to TextArea. I think we should just v1 this.

Reply:
Kk that's fine with me, will update now